### ISSUE TITLE
hx-post with external submit button excludes the button value

### ISSUE THREAD

**Issue by wooooodward** [bigskysoftware/htmx#1120]
Submitting a form with the submit button outside of the `<form>` tag does not include the button's `name` attribute as a parameter or its value (`'lol': 'cats'`) in the request which is inconsistent with the default expected behaviour.

```
<form id="my-form" hx-post="/example">
  <input type="text" name="name" />
</form>

<input type="submit" form="my-form" name="lol" value="cats"/>
```

**Comment by mihalikv** [comment]
Same issue here with the newest version 1.8.5.

**Comment by drowe-official** [comment]
Also ran into this issue. It popped up when I was trying to use form buttons inside a Bootstrap modal footer. For anyone in the same boat- I ended up having to wrap the entire `modal-dialog` div in the `form` tag, which isn't ideal.

**Comment by alexpetros** [comment]
Resolved by #1559
